.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,11 @@
+name: Integration Tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@enable-ingress-integration-tests
```

Comment:
This is not pointing to a release. I assume you'll want to change it before merging

Reply:
Yes, just need that PR to enable ingress to be merged 😄

Reply:
It has been merged :)

Reply:
Thank you, switched this to point at main now